post/release-0-2.qmd

```diff
@@ -0,0 +1,433 @@
+- Authority code example:
+
+  ```json
+  {
+      "crs": "EPSG:32620",
+      "crs_type": "authority_code",
+  }
+  ```
+
+  Note that this should only be used as a last resort. Producers should prefer writing a complete description of the CRS.
+
+- SRID example:
+
+  ```json
+  {
+      "crs": "32620",
+      "crs_type": "srid",
+  }
+  ```
+
+  Note that this should only be used as a last resort for database drivers or readers that have no other option.
+
+- Opaque CRS example:
+
+  ```json
+  {
+      "crs": 'PROJCS["WGS 84 / UTM zone 20N",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",-63],PARAMETER["scale_factor",0.9996],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH],AUTHORITY["EPSG","32620"]]',
+  }
+  ```
+
+  Opaque CRS strings should only be used as a last resort when the producer has no way to communicate more information. In this case, the CRS is a WKT1 string, which doesn't fit into any other allowed variants.
+
+### Newly-supported data types for WKT and WKB arrays
+
+Version 1.4 of the Arrow Columnar format added ["binary view"][binary-view-spec] and "string view" layouts. This new representation for an array of strings or bytes [is][string-view-ref-1] [widely][string-view-ref-2] [held][string-view-ref-3] [to be][string-view-ref-4] more performant for common use cases.
+
+GeoArrow 0.2 allows serialized Well-Known Binary (WKB) and Well-Known Text (WKT) arrays to be backed by these new binary view and string view data types.
+
+[binary-view-spec]: https://arrow.apache.org/docs/format/Columnar.html#variable-size-binary-view-layout
+[string-view-ref-1]: https://cedardb.com/blog/german_strings/
+[string-view-ref-2]: https://datafusion.apache.org/blog/2024/09/13/string-view-german-style-strings-part-2/
+[string-view-ref-3]: https://engineering.fb.com/2024/02/20/developer-tools/velox-apache-arrow-15-composable-data-management/
+[string-view-ref-4]: https://pola.rs/posts/polars-string-type/
+
+### Edge interpolations
+
+Second, the `"edges"` parameter, which could previously have been omitted (to indicate
+planar/linear edges as defined by simple features) or taken the value `"spherical"`
+to indicate that vertices should be connected by approximating the edge as the
+shortest path on a perfect sphere. In the recently voted in
+[GEOMETRY and GEOGRAPHY types in Parquet and Iceberg](https://cloudnativegeo.org/blog/2025/02/geoparquet-2.0-going-native/),
+four other options (`"vincenty"`, `"thomas"`, `"andoyer"`, and `"karney"`) were
+added to ensure GeoArrow can faithfully represent the intent of an arbitrary Parquet
+and/or Iceberg source.
```

Comment:
Also here there is something wrong with the sentence? It's Parquet that has those options, so we added them to GeoArrow as well